frontend/awx/administration/instances/InstanceDetails.cy.tsx

```diff
@@ -0,0 +1,56 @@
+import { formatDateString } from '../../../../framework/utils/formatDateString';
+import { capitalizeFirstLetter } from '../../../../framework/utils/strings';
+import { Instance } from '../../interfaces/Instance';
+import { InstanceDetails } from './InstanceDetails';
+
+describe('Mesh Visualizer', () => {
+  beforeEach(() => {
+    cy.intercept({ method: 'GET', url: '/api/v2/instances/*' }, { fixture: 'instance.json' }).as(
+      'getInstance'
+    );
+    cy.intercept(
+      { method: 'GET', url: '/api/v2/instances/*/instance_groups/' },
+      { fixture: 'instance_groups.json' }
+    ).as('getInstanceGroups');
+  });
+
+  it('Renders name, node type, node status, instance groups, download bundle, used capacity, running jobs, total jobs, policy type, memory, last health check, created, modified, forks, and enabled fields', () => {
+    cy.mount(<InstanceDetails />);
+    cy.wait('@getInstance')
+      .its('response.body')
+      .then((instance: Instance) => {
+        cy.get('[data-cy="name"]').should('have.text', instance.hostname);
+        cy.get('[data-cy="node-type"]').should(
+          'have.text',
+          `${capitalizeFirstLetter(instance.node_type)}`
+        );
+        cy.get('[data-cy="status"] .pf-v5-c-label__text').should(
```

Comment:
is there a possible alternative to the patternfly locator?

Reply:
Updated our `StatusLabel` component to take a `dataCy` prop. 👍